### ISSUE TITLE
Can't find the form to Download the dataset

### ISSUE THREAD

**Issue by Luffy2810** [woodfrog/floor-sp#34]
When I try to open the link, a website opens but this website has no form and looks like a product website.
If there exists a form link in the website, I can't find it.
Can you please help me with it

**Comment by woodfrog** [comment]
Indeed, the original page of the dataset seems to be offline now, probably due to some changes made by Beike. Since we do not have the right to distribute Beike's data, can you try to contact them to see if they can provide new download links? Thanks
